docs/resources/resources-links-help.md

```diff
@@ -1,25 +1,76 @@
 ---
 title: Office Add-ins additional resources
 description: Useful resources about Office Add-ins.
-ms.date: 04/14/2021
+ms.date: 02/22/2023
 ms.localizationpriority: medium
 ---
 
 # Office Add-ins additional resources
 
 These resources provide additional information and support for developing Office Add-ins.
 
-|**Resource type**|**Guidance**|
+## Product support and service issues
+
+Use **Azure Admin Center** or **Microsoft 365 Admin Center** for any **business-critical issues** that need SLA-based support.
+
+| **Product support** | **Contact** |
```

Comment:
I don't think you need to include the bold formatting in the top row as it's bolded by the platform

```suggestion
| Product support | Contact |
```